src/pipecat/runner/run.py

```diff
@@ -177,7 +182,12 @@ def _create_server_app(
     return app
 
 
-def _setup_webrtc_routes(app: FastAPI, esp32_mode: bool = False, host: str = "localhost"):
+def _setup_webrtc_routes(
+    app: FastAPI,
+    esp32_mode: bool = False,
+    host: str = "localhost",
+    prebuilt_ui_config: SmallWebRTCPrebuiltArguments = None,
```

Comment:
I guess in this case it should be Optional.
```suggestion
    prebuilt_ui_config: Optional[SmallWebRTCPrebuiltArguments] = None
```